### ISSUE TITLE
Use Gemmi to interconvert between pdb and cif?

### ISSUE THREAD

**Issue by danny305** [Electrostatics/pdb2cif#1]
[Gemmi](https://github.com/project-gemmi/gemmi) is an open sourced c++ library with python bindings for working with structural biology data. 

I have implemented code to convert a pdb to a cif with gemmi (in python) in the past (screenshot below) and I am pretty sure its probably just as simple to go from cif to pdb.
 
<img width="475" alt="image" src="https://user-images.githubusercontent.com/31376682/116498168-bc108400-a86e-11eb-80d1-b7315e47c4be.png">

So I am currently finishing up implementing cif functionality ([reading](https://github.com/mittinatten/freesasa/pull/51) and [writing](https://github.com/mittinatten/freesasa/pull/59)) to the [freesasa](https://github.com/mittinatten/freesasa) C library with the author of the repository and we just used gemmi for everything. We have freesasa outputting a freesasa cif file where we appended cif blocks specific to freesasa and added 2 additional columns to the `_atom_site.` block. 1  the freesasa solvent accessibility value and the other for freesasa atomic radii. This is kinda what I was thinking about doing with pdb2pqr....?

**Comment by sobolevnrm** [comment]
Hi Danny -- thanks for the suggestion.  We're trying to favor native Python solutions as much as possible given problems our user base has had in the past with compilers.  Even if it duplicates effort a bit, I'd prefer to stick with a native Python solution for this interconversion.

**Comment by danny305** [comment]
I agree with that approach and I honestly prefer it. Working with the C++ gemmi api was definitely a learning curve. 

However, we can use gemmi 100% in python. Using gemmi in this situation is not much different to using numpy (where everything is implemented in C/C++ and then wrapped with python). Gemmi has a nice python api, python documentation, and can be installed with `pip install gemmi`, removing all compiler issues. 

This can reduce a lot of the low level code duplication. Gemmi cif parser is very robust and I have had zero trouble working with them (I cannot say the same about pdb2pqr cif reading capability--its very finicky, especially with pdb/cif files from the pdb-redo server.)

If you still rather not use gemmi I understand! I am just providing as much information so y'all make the best decision for pdb2pqr.
